misc/output_test.py

```diff
@@ -174,11 +174,11 @@ def test_issue_1418(self) -> None:
   description = echo $out
 
 build a: echo
-  delay = 3
+  delay = 0.9
 build b: echo
-  delay = 2
+  delay = 0.6
 build c: echo
-  delay = 1
+  delay = 0.3
```

Comment:
I suggest leaving this out of this PR, it's unrelated to what you're trying to do. Unfortunately, we had issues in the past because CI tests are run on VMs that often are very contended, meaning that tests that expect exact timing sequences can fail randomly in Github actions :-/ I am wary that such a change might trigger this condition more often.

Reply:
Sure, I will drop it on the following rebase.